app/views/root/campaign.html.erb

```diff
@@ -1,52 +1,25 @@
 <% content_for :title, page_title(@publication) %>
+<div class="header-context group"><!-- deliberately empty --></div>
```

Comment:
Would it be possible to extend this `deliberately empty` comment to explain why it is deliberately empty please.